src/test/resources/regressions/features/trusted/trusted-functions.gobra

```diff
@@ -25,8 +25,18 @@ func positiveSumWithoutTrusted(x int, y int) (ret int) {
 
 trusted
 func functionWithNonSupportedFeature() int{
-	some non supported go feature
-	return 0
+	// fallthrough statements would normally cause a translation error
+	// since they aren't supported
```

Comment:
I would rely here on some feature that Gobra does not (yet) support. Couldn't you write a body that would cause a type error if it would be type checked? E.g. assigning something to an incompatible type